packages/sync/src/class-lock.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Lock class.
+ *
+ * @package automattic/jetpack-sync
+ */
+
+namespace Automattic\Jetpack\Sync;
+
+/**
+ * Lock class
+ */
+class Lock {
+	/**
+	 * Prefix of the blog lock transient.
+	 *
+	 * @access public
+	 *
+	 * @var string
+	 */
+	const LOCK_PREFIX = 'jp_sync_lock_';
+
+	/**
+	 * Default Lifetime of the lock.
+	 *
+	 * @access public
+	 *
+	 * @var int
+	 */
+	const LOCK_TRANSIENT_EXPIRY = 15; // Seconds.
+
+	/**
+	 * Attempt to lock.
+	 *
+	 * @access public
+	 *
+	 * @param string $name lock name.
+	 * @param int    $expiry lock lifetime.
```

Comment:
Instead of lifetime can we call it duration of the lock in seconds. I think that would make things more clear.

Reply:
🤷‍♀ 
```suggestion
	 * @param int    $expiry lock duration.
```